src/controllers/dlgprefcontroller.cpp

```diff
@@ -123,7 +134,7 @@ void DlgPrefController::showLearningWizard() {
     slotApply();
 
     // After this point we consider the mapping wizard as dirtying the preset.
-    slotDirty();
+    setDirty(true);
```

Comment:
I am confused here. Is this dirtying DlgPrefController or the preset? It is not obvious because they both get called "dirty".

Reply:
dirtying the preset would be `m_pPreset->setDirty(true)` but i can add a comment

Reply:
I think a better solution would be to not use the word "dirty" and use words that convey what is happening.

Reply:
Huh, no wonder you were confused: The line is wrong there. I removed it. The dirty state will be correctly set on the preset itself if anything has been learned by the wizard.